guides/common/modules/proc_enabling-the-installer-managed-dns-service.adoc

```diff
@@ -0,0 +1,52 @@
+[id="enabling-the-installer-managed-dns-service_{context}"]
+= Enabling the installer-managed DNS service
+
+If you do not have a DNS server available in your network, you can use the installer-managed DNS service. This feature enables you to provide a DNS service with a low maintenance effort.
+
+You can use a backup of the answer file that was created before configuring external DNS, or you can create a backup of the answer file.
+ifndef::orcharhino[]
+For more information about answer files, see {InstallingServerDocURL}configuring-server_{project-context}[Configuring {ProjectServer}].
+endif::[]
+
+Complete the steps on the {Project} or {SmartProxyServer} that you want to configure to manage DNS service for the domain.
+
+
+.Procedure
+
+. Configure {Project} or {SmartProxy} as a DNS server:
+
+** If you have a backup of the answer file that was created before configuring a non-installer-managed DNS service:
+
+... Restore the answer file.
+
+... Re-apply the answer file:
++
+[options="nowrap",subs="+quotes,attributes"]
+....
+# {foreman-installer}
+....
+
+** If you do not have a backup answer file that was created before configuring a non-installer-managed DNS service:
+
+... Create a backup of the answer file now.
+
+... Configure {Project} or {SmartProxy} as DNS server:
++
+[options="nowrap",subs="+quotes,attributes"]
+....
+# {foreman-installer} \
+--foreman-proxy-dns-managed=true \
+--foreman-proxy-dns-provider=nsupdate \
+--foreman-proxy-dns-server="127.0.0.1" \
+--foreman-proxy-dns=true
+....
++
+ifeval::["{context}" == "{smart-proxy-context}"]
+For more information, see xref:configuring-dns-dhcp-and-tftp-on-productname_{smart-proxy-context}[].
+endif::[]
+ifeval::["{context}" == "{project-context}"]
+For more information, see {InstallingSmartProxyDocURL}configuring-dns-dhcp-and-tftp-on-productname_{smart-proxy-context}[Configuring DNS, DHCP, and TFTP on {SmartProxyServer}].
+endif::[]
```

Comment:
And similar to the DHCP one: I'd expect us to drop those sections from the installation manuals and drop this too.
```suggestion
```